frontend/src/component/project/Project/ProjectInfo/ProjectInfo.tsx

```diff
@@ -50,8 +51,11 @@ const ProjectInfo = ({
     const { uiConfig, isEnterprise } = useUiConfig();
     const theme = useTheme();
     const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
+    const hasChangeRequestsEnabled = useChangeRequestConfig(id).data?.some(
```

Comment:
There's a hook useChangeRequestsEnabled.ts that returns isChangeRequestConfiguredInAnyEnv